src/drivers/Kconfig

```diff
@@ -6,6 +6,8 @@ rsource "intel/Kconfig"
 
 rsource "dw/Kconfig"
 
+rsource "imx/Kconfig"
```

Comment:
Should I add a submenu that only shows on i.MX platforms for these drivers?

Reply:
@paulstelian97 yes, please. Look at how src/drivers/intel is handling this thing.

Reply:
I don't see anything on the Intel side for this actually. Is it possible that menuconfig is automatically hiding the unsatisfiable options?

Reply:
I have added a small guard similar to what DW did.

Reply:
@paulstelian97 although we have this non-conditional include for Intel

```
rsource "intel/Kconfig"
```

I think you do not see anything on this Intel side because for IMX none of the Intel symbols are selected. That's just a guess I don't know how this actually works.

So, split the patches like this:

1) patch 1, adds:

```rsource "imx/Kconfig"
```
and moves iMX_EDMA to src/drivers/imx/Kconfig

2) patch 2, adds the rest of missing config symbols

Reply:
I do not see anything from the Intel side because the options' "depends on" is missing. If I activate "show all" I do see the Intel options in red (and they're only missing because of the unsatisfied dependency on INTEL). I will rely on this for the i.MX option. But my question was whether I should also group them into a submenu or not.

Reply:
Not sure what would be de difference. Do you mean like, when the user goes to Drivers there will be a sub-menu like IMX drivers and then the actual options will be available?

Just go without a submenu for now, we will figure out later what;s the optimal solution.